Private/Find-ESC1.ps1

```diff
@@ -83,6 +83,11 @@ Get-ADObject `$Object | Set-ADObject -Replace @{'msPKI-Enrollment-Flag' = 0}
 "@
                     Technique             = 'ESC1'
                 }
+
+                if ( $Mode -in @(1, 3, 4) ) {
+                    Update-ESC1Remediation -Issue $Issue
```

Comment:
Haven't tested, but I feel like you need to capture the updated remediation in $Issue before continuing to return $Issue at the end of the function. (And I could be totally wrong.) Should this be something like:

```powershell
if ( $Mode -in @(1, 3, 4) ) {
    $Issue = Update-ESC1Remediation -Issue $Issue
}

$Issue
```